### PR TITLE
👷 add lint + ci

### PR DESCRIPTION
**Pull request by bcaudan**
# Motivation

ensure that PRs updating the schema pass the validation

# Changes

* add lint: [jsonlint package](https://github.com/zaach/jsonlint) seems maintained and easy to setup
* add ci through github actions: easy to setup and used in other public repositories

# Test

cf github action checks

### CODE REVIEW

**Review by xgouchet — APPROVED** [review]

**Review by ncreated — APPROVED** [review]
Nice 💪

**Review by amortemousque — APPROVED** [review]

File: lint.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+line='----------------------------------------'
+
+for file in `ls *.json && ls */*.json`; do
```
Comment (BenoitZugmeyer) [inline lint.sh]: **💬 suggestion:** ‏

```suggestion
shopt -s globstar
for file in **/*.json; do
```

**Review by BenoitZugmeyer — APPROVED** [review]

File: lint.sh (hunk 1 above)
Comment (bcaudan) [reply lint.sh]: sweet 🤩

Comment (BenoitZugmeyer) [reply lint.sh]: Or maybe simply use git, since it knows which json files are tracked:
```suggestion
for file in `git ls-files '*.json'`; do
```

Comment (bcaudan) [reply lint.sh]: the first one is closer to what I was going for, I went with it

**Pull request merged**